src/StoreApi/Routes/V1/Patterns.php

```diff
@@ -104,13 +105,24 @@ protected function get_route_post_response( \WP_REST_Request $request ) {
 		$business_description = sanitize_text_field( wp_unslash( $request['business_description'] ) );
 		$vertical_id          = ( new VerticalsSelector() )->get_vertical_id( $business_description );
 
+		if ( empty( $business_description ) ) {
+			$business_description = get_option( 'woo_ai_describe_store_description' );
```

Comment:
If this option is empty and returns `false` and `get_option( 'last_business_description_with_ai_content_generated' )` is also empty (and returns `false`) in `ProductUpdater->generate_content()` the following will evaluate as `true` and return early:

`$last_business_description === $business_description`

Is there a danger of that, and would that be expected?

Reply:
> Is there a danger of that, and would that be expected?

It wouldn't be dangerous; the consequence here is that we would bail early with a success message instead of an error. Thanks for flagging this; nice catch! Feedback addressed on https://github.com/woocommerce/woocommerce-blocks/pull/11155/commits/3da76eb5ef1ea018cb9f6de71a1b5a2cfabba7cd